python/fledge/services/core/api/notification.py

```diff
@@ -58,10 +54,36 @@ async def get_plugin(request):
     except Exception as ex:
         msg = str(ex)
         _logger.error(ex, "Failed to get notification plugin list.")
-        raise web.HTTPInternalServerError(reason=msg, body=json.dumps({"message": msg}))
+        raise ValueError(msg)
     else:
-        return web.json_response({'rules': rule_plugins, 'delivery': delivery_plugins})
+        resp = {}
+        if rule_plugins is not None:
+            if 'rules' not in rule_plugins:
+                resp['rules'] = rule_plugins
+            else:
```

Comment:
What does this mean? Are we not sure about the  response object of rule_plugins from notification service?

Reply:
@pintomax could you please clarify this

Reply:
OK this is array vs dict object